### ISSUE TITLE
[QUESTION] Can you provide precompiled libSqliteICU.so?

### ISSUE THREAD

**Issue by malloth** [requery/sqlite-android#30]
Can you provide (in github's releases) libSqliteICU.so precompiled for the same architectures as your SQLite-android lib? I'm asking because I'd like to use SQLiteCustomExtension to load ICU extension (as you don't give a direct support for LOCALIZED collation).

Alternatively, can you add a full support for LOCALIZED collator in future releases?
This is very important as BINARY sorting and grouping is not an option for my country locale.

**Comment by IlyaDvoinoi** [comment]
> Alternatively, can you add a full support for LOCALIZED collator in future releases
+1

**Comment by npurushe** [comment]
Yes this is something that could be included as a separate artifact. It was not included originally due to the size of the library
